Documentation/cloud-config-deprecated.md

```diff
@@ -1,5 +1,14 @@
 # Deprecated Cloud-Config Features
 
+---
+
+**NOTE**: This project has been superseded by [Container Linux Configs][cl-config] and [Ignition][ignition]. CoreOS Cloud-Config is no longer under active development. Please direct all development efforts to Ignition.
+
+[ignition]: https://github.com/coreos/ignition
+[cl-config]: https://coreos.com/matchbox/docs/latest/container-linux-config.html
```

Comment:
This is an odd document to link. The provisioning docs mentioned above are the canonical source for information about this topic.